src/hotspot/share/runtime/synchronizer.cpp

```diff
@@ -130,12 +142,8 @@ size_t MonitorList::unlink_deflated(Thread* current, LogStream* ls,
       m = m->next_om();
     }
 
-    if (current->is_Java_thread()) {
-      // A JavaThread must check for a safepoint/handshake and honor it.
-      ObjectSynchronizer::chk_for_block_req(JavaThread::cast(current), "unlinking",
-                                            "unlinked_count", unlinked_count,
-                                            ls, timer_p);
-    }
+    // Must check for a safepoint/handshake and honor it.
+    safepointer->block_for_safepoint("unlinking", "unlinked_count", unlinked_count);
```

Comment:
At the top of this code block:
```
L108  // The in-use list head can be null during the final audit.
L109  while (m != nullptr) {
```
We're no longer deflating during the final audit right? So the comment
on L108 can be deleted.